book/chrome.md

```diff
@@ -1395,9 +1395,7 @@ address bar. Honestly, do this exercise just for your sanity.
 to open it in a new tab. You might want to use a mouse when testing.
 
 7-3 *Window title*. Browsers set their window title to the contents of the
-current tab's `<title>` element. Make your browser do the same. You
-can call the `title` method of your browser's `window` field to change
```

Comment:
This seems to be a request to add a JS feature? Which we don't have as of chapter 7.

Reply:
Ah, no, this isn't what this text is about. (On the web it's `document.title`.) It is indeed bad writing, but it's saying that there's a `Tk.title` method which you can call on your `Browser.window` field to set the window title.